kotlin-libraries/src/test/kotlin/com/baeldung/mockk/AnnotationMockKUnitTest.kt

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.mockk
+
+import io.mockk.*
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.impl.annotations.RelaxedMockK
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class InjectTestService {
+    lateinit var service1: TestableService
+    lateinit var service2: TestableService
+
+    fun invokeService1(): String {
+        return service1.geDataFromDb("Test Param")
+    }
+}
+
+class AnnotationMockKUnitTest {
+
+    @MockK
+    lateinit var service1: TestableService
+
+    @RelaxedMockK
+    lateinit var service2: TestableService
+
+    @InjectMockKs
+    var objectUnderTest = InjectTestService()
+
+    @BeforeEach
+    fun setUp() = MockKAnnotations.init(this)
+
+    @Test
+    fun givenServiceMock_whenCallingMockedMethod_thenCorrectlyVerified(){
+        // given
+        every { service1.geDataFromDb("Test Param") } returns "No"
+        // when
+        val result = objectUnderTest.invokeService1()
+        // then
+        assertEquals(result, "No")
```

Comment:
The order of params in assertEquals should be reversed in all the tests, so that it's assertEquals(expected, actual)

Reply:
Good point. I use AssertJ for assertions, so I used to put the expected value at the end 🙃 
Fixed